mobile/test/cc/unit/envoy_config_test.cc

```diff
@@ -38,7 +38,7 @@ TEST(TestConfig, ConfigIsApplied) {
       .addStatsFlushSeconds(654)
       .setAppVersion("1.2.3")
       .setAppId("1234-1234-1234")
-      .enableDnsCache(true)
+      .enableDnsCache(true, /* save_interval_seconds */ 1)
```

Comment:
Since 1 is the default, it would probably be a good idea to select a non-default value.

Reply:
Yes, although I think something is broken here, as I appear to be getting a yaml comparison error when anything other than the default value is specified here:

```
[2023-02-03 15:28:27.464][5642470][critical][assert] [library/cc/engine_builder.cc:961] assert failure: generatedStringMatchesGeneratedBoostrap(generateConfigStr(), *bootstrap). Details: Native C++ checks failed
```

Reply:
Ordering was wrong.